dockstore-yaml-proposals/filebrowser/docker-compose.yaml

```diff
@@ -7,7 +7,7 @@ services:
   entrypoint:
    - /bin/sh
    - -c
-   - /filebrowser config init &&  /filebrowser config set --auth.method=noauth && /filebrowser --noauth --root="/home" --address=0.0.0.0 --database=/config/filebrowser.db
+   - rm -rf /home/${USER_NAME}/filebrowser.db && /filebrowser --noauth --root="/home" --address=0.0.0.0 --database=/home/${USER_NAME}/filebrowser.db
```

Comment:
Are there any filebrowser settings that should be kept between sessions?  I guess it's ok to clear it out before every session, but maybe nice to keep it if there are settings that a user might want to keep between sessions.